SoftLayer/CLI/dedicatedhost/detail.py

```diff
@@ -0,0 +1,71 @@
+"""Get details for a dedicated host."""
+# :license: MIT, see LICENSE for more details.
+
+import logging
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer import utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--price', is_flag=True, help='Show associated prices')
+@click.option('--guests', is_flag=True, help='Show guests on dedicated host')
+@environment.pass_env
+def cli(env, identifier, price=False, guests=False):
+    """Get details for a virtual server."""
+    dhost = SoftLayer.DedicatedHostManager(env.client)
+
+    table = formatting.KeyValueTable(['name', 'value'])
+    table.align['name'] = 'r'
+    table.align['value'] = 'l'
+
+    result = dhost.get_host(identifier)
+    result = utils.NestedDict(result)
+
+    table.add_row(['id', result['id']])
+    table.add_row(['name', result['name']])
+    table.add_row(['cpu count', result['cpuCount']])
+    table.add_row(['memory capacity', result['memoryCapacity']])
+    table.add_row(['disk capacity', result['diskCapacity']])
+    table.add_row(['create date', result['createDate']])
+    table.add_row(['modify date', result['modifyDate']])
+    table.add_row(['router id', result['backendRouter']['id']])
+    table.add_row(['router hostname', result['backendRouter']['hostname']])
+    if utils.lookup(result, 'billingItem') != {}:
+        table.add_row(['owner', formatting.FormattedItem(
+            utils.lookup(result, 'billingItem', 'orderItem',
```

Comment:
wouldn't this lookup just return {} if billingItem wasn't set, so the if above isn't needed? Because either way if something in that lookup isn't set, you'll end up with `formatting.blank()` anyways